### PR TITLE
Adds post on zero downtime migrations

### PR DESCRIPTION
**Pull request by sourya**
Signed-off-by: Sourya <souryavatsyayan@gmail.com>

### CODE REVIEW

**Comment by dolftax** [comment]
@sourya 

Thanks for the article.

- Could you add some pointers on different cases from here? - 
http://souryavatsyayan.com/2018/09/21/zero-downtime-migrations.html
- It would be great if you can add some example pseudocode of the checks

**Comment by sourya** [comment]
Made the changes. Please take a look.

**Comment by sourya** [comment]
screencapture-localhost-1313-2018-10-zero-downtime-migrations-2018-10-30-21_23_22
This is how it looks.

**Review by dolftax — DISMISSED** [review]

File: content/article/zero-downtime-migrations.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+type: "article"
+---
+
+## Introduction
+
+Say you are maintaining an e-commerce application that has a catalog of 100 products stored in a table named `product`
+in an RDBMS database, say PostgreSQL. Now, let's add a new column named `discount` to the table.
+Let's use this query:
+
+`ALTER TABLE product ADD COLUMN "discount" INTEGER NOT NULL DEFAULT 0;`
```
Comment (sanketsaurav) [inline content/article/zero-downtime-migrations.md]: For syntax highlighting of this SQL query, add the code block in this way instead:

```
    ```sql
     ALTER TABLE product ADD COLUMN "discount" INTEGER NOT NULL DEFAULT 0;
     ```
```

Please do this for all other code blocks.

**Review by sanketsaurav — CHANGES_REQUESTED** [review]
@sourya Please rebase with the current master and push.

File: content/article/zero-downtime-migrations.md (hunk 1 above)
Comment (sourya) [reply content/article/zero-downtime-migrations.md]: Done.

**Comment by sanketsaurav** [comment]
@sourya Thanks for your contribution. Your article is live [here](https://deepthought.blog/2018/10/zero-downtime-database-migration/). 🎊

**Comment by sourya** [comment]
Thank you.

**Pull request merged**